### DISCUSSION TITLE
Can grafana set to disable admin login?

### DISCUSSION THREAD

**Discussion by mmt-10** [grafana/grafana#44958] [General]
Hi,
I would like to set to disable admin login.
How do I set option to grafana.ini?

**Comment by a-imran99** [comment]
https://stackoverflow.com/questions/33111835/how-to-set-up-grafana-so-that-no-password-is-necessary-to-view-dashboards

may be this could help.

**Comment by mmt-10** [comment]
@a-imran99 

Thank you for your reply.
Grafana is opened to Internet.
I would not like to use anonymous access.
I make viewing user and would like to use auth.
So, I would like to set to disable admin login.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
